packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -486,13 +488,18 @@ export default class ImageEdit implements EditorPlugin {
                 if (context?.elementClass == ImageEditElementClass.ResizeHandle) {
                     const clientWidth = wrapper.clientWidth;
                     const clientHeight = wrapper.clientHeight;
+                    const wasResized = this.image.getAttribute('wasResized');
                     doubleCheckResize(
                         this.editInfo,
                         this.options.preserveRatio,
                         clientWidth,
                         clientHeight
                     );
 
+                    if (!wasResized) {
+                        this.image.setAttribute('wasResized', 'true');
```

Comment:
we already have ImageEditInfo object, so no need to add one more attribute to the image. You just need to add one more property to ImageEditInfo and save this result there.

Reply:
Oh, I understood, I'll fix this.